src/EchoKeyDynamicFunctionReturnTypeExtension.php

```diff
@@ -47,106 +45,109 @@ class EchoKeyDynamicFunctionReturnTypeExtension implements \PHPStan\Type\Dynamic
     ];
 
     /**
-     * Functions with strictly boolean `echo` key.
+     * Functions that do not accept a query string.
      */
-    private const STRICTLY_BOOL = [
+    private const STRICTLY_ARRAY = [
         'get_search_form',
+        'wp_login_form',
+    ];
+
+    /**
+     * Function that can return void if echo is not true/truthy.
+     */
+    private const ALWAYS_VOID = [
         'the_title_attribute',
-        'wp_list_authors',
+        'wp_dropdown_languages',
+        'wp_get_archives',
         'wp_list_comments',
-        'wp_list_pages',
-        'wp_list_users',
-        'wp_login_form',
-        'wp_page_menu',
```

Comment:
Where have these removed ones go?

Reply:
I removed the `STRICLTY_BOOL` functions because I dropped the check for int values of echo, `wp_dropdown_languages`, `wp_get_archives` and `wp_list_comments` are now part of `ALWAYS_VOID`.